### PR TITLE
feat(Menu): Expose function to close menu

### PR DESCRIPTION
**Pull request by andyindahouse**
https://user-images.githubusercontent.com/4282123/127829778-0baeb0fb-220d-4b82-abca-e16edf28d39d.mov

### CODE REVIEW

File: src/menu.tsx [hunk 1]
```diff
@@ -40,6 +41,7 @@ type TargetRenderProps = {
 };
 
 export type MenuProps = {
+    open: boolean;
```
Comment (pladaria) [inline src/menu.tsx:44]: where are you using this?

Comment (andyindahouse) [reply src/menu.tsx]: leftovers

File: src/__stories__/menu-story.tsx [hunk 2]
```diff
@@ -49,6 +58,38 @@ export const Default: StoryComponent = () => {
...
+                        {fruitOptions.map((fruit) => (
+                            <Box paddingX={16} key={fruit.text} paddingY={8}>
+                                <Touchable
+                                    onPress={() => {
+                                        close();
+                                        confirm({
+                                            title: 'Title',
+                                            message: 'This is the dialog message',
+                                            icon: <IcnInfo />,
+                                        });
```
Comment (sergivillar) [inline src/__stories__/menu-story.tsx:83]: why?

File: src/__stories__/menu-story.tsx [hunk 3]
```diff
@@ -39,7 +41,14 @@ export const Default: StoryComponent = () => {
                             <Box paddingX={16} paddingY={8}>
                                 <Checkbox
                                     name={fruit.text}
-                                    onChange={() => setValues(fruit.value)}
+                                    onChange={() => {
+                                        confirm({
+                                            title: 'Title',
+                                            message: 'This is the dialog message',
+                                            icon: <IcnInfo />,
+                                        });
```
Comment (sergivillar) [inline src/__stories__/menu-story.tsx:49]: why?

File: src/__stories__/menu-story.tsx (hunk 2 above)
Comment (andyindahouse) [reply src/__stories__/menu-story.tsx:83]: @pladaria ask me to add a story with the use of this new prop

Comment (sergivillar) [reply src/__stories__/menu-story.tsx:83]: yes, I'm talking about show a modal that will freeze the UI each time I'm playing with Menu component

File: src/__stories__/menu-story.tsx (hunk 3 above)
Comment (andyindahouse) [reply src/__stories__/menu-story.tsx]: removed, here doesn't make sense this

File: src/__stories__/menu-story.tsx (hunk 2 above)
Comment (andyindahouse) [reply src/__stories__/menu-story.tsx:78]: you can already check the last version deployed https://mistica-web-4s5z16vd2-atabel.vercel.app/?path=/story/components-controls-menu--default

**Review by pladaria — APPROVED** [review]

**Review by sergivillar — APPROVED** [review]

**Pull request merged**